static/resources/yaml/observability_pipelines/splunk/aws_eks.yaml

```diff
@@ -0,0 +1,132 @@
+datadog:
+  apiKey: "<datadog_api_key>"
```

Comment:
Is it best for our customers to be hardcoding API keys and Splunk tokens in their config? Wondering what the pros/cons are of this approach vs. the existing guidance in the UI:

```
helm install observability-pipelines-worker \
    --set datadog.site=datadoghq.com \
    --set datadog.apiKey= \
    --set datadog.pipelineId=f1d9be60-e5b9-11ec-b1d4-da7ad0900002 \
    --set datadog.remoteConfigurationEnabled=true \
datadog/observability-pipelines-worker
```